tweets/2022-04/postmeetup9.tweet

```diff
@@ -0,0 +1 @@
+In case you missed this week's meetup, catch @DuggalAakanksha,@irlNnContainer,@oindrilla_chat, & @Shreyvert talk about #AI4CI that uses #open ops data from @kubernetesio & @openshift #CI/CD platforms & check the ML services deployed on @OperateFirst🚀See you at the next meetup!
```

Comment:
trying to get creative here with the word limit. Does this work? 😃 
```suggestion
In case you missed this week's meetup, catch @DuggalAakanksha,@irlNnContainer,@oindrilla_chat, & @Shreyvert talk about #AI4CI ops tool using @kubernetesio & @openshift #CI/CD data & check the ML services on @OperateFirst! https://youtu.be/M4thSVYdIT0 See you at the next meetup! 🚀
```